docs/research/CLAUDE_STYLE_RULE_FOLLOWING_RECORDS.md

```
@@ -0,0 +1,420 @@
+| parking | tells | X | rules_v2_ex | 8 | 9 | 9 |
+| parking | tells | Y | rules_v2 | 6 | 8 | 8 |
+| parking | colleague | X | rules_v2_ex | 7 | 8 | 9 |
+| parking | colleague | Y | rules_v2 | 5 | 6 | 8 |
+
+## Round 1 drafts and fact briefs
+
+### Scenario `batch`
+
+Fact brief:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)